src/Handler/UpdatePostHandler.php

```diff
@@ -32,9 +32,9 @@ public function run(): Response
         // TODO
         if (count($error_list) > 0) return new Response(status_code: OK_STATUS_CODE, html: "<div>更新に失敗しました。</div>");
 
-
-        $dto = new UpdatePostDto(title: $title, body: $body, thumbnail_id: 1);
-        $this->post_client->updatePost($this->post_id, $dto);
+        // TODO: editable thumbnail, tags, images
+        $dto = new UpdatePostDto(title: $title, body: $body);
+        $this->post_repo->updatePost($this->post_id, $dto);
 
         $redirect_url = "http://localhost:8080/posts/" . $this->post_id;
```

Comment:
```suggestion
        $redirect_url = HOST_BASE_URL . $this->post_id;
```
ここもせっかくならHOST_BASE_URLを使いましょうか。